src/main/flight/pid.h

```diff
@@ -206,10 +204,13 @@ typedef struct pidProfile_s {
     uint8_t dyn_idle_d_gain;                // D gain for corrections around rapid changes in rpm
     uint8_t dyn_idle_max_increase;          // limit on maximum possible increase in motor idle drive during active control
 
+    uint8_t feedforwardTransition;          // Feedforward attenuation around centre sticks
```

Comment:
This should go at the end of the parameter group. Also, since this is an array type parameter group, the version number will have to be incremented in https://github.com/betaflight/betaflight/pull/10777/files#diff-afd1226d5a8ae512e5fc53a545d9b7259b4f001bc94eae9c38cacd8762fb7f54R124.

Reply:
I am probably wrong, but for readability and maintenance, I thought it might help to keep all related parameters together.